consumer-backend/consumer.js

```diff
@@ -5,9 +5,10 @@ const express = require('express');
 const ws = require('ws');
 const app = express();
 const serviceBindings = require('kube-service-bindings');
+const { oauthBearerProvider } = require('./utils.js');
 
 const topic = process.env.KAFKA_TOPIC || 'countries';
-const groupId = process.env.KAFKA_GROUP_ID || 'consumer-test';
+const groupId = process.env.KAFKA_GROUPID || 'consumer-test';
```

Comment:
This could break people of have already used the example ?  Not sure if that is a concern, but maybe we should just leave it as is?

Reply:
> This could break people of have already used the example ? Not sure if that is a concern, but maybe we should just leave it as is?

This variable is introduced on this PR, so nobody has used it so far. I had it wrong on readme file on a previous commit of this PR and decided to fix it on the code instead of fixing it on the readme file.